### ISSUE TITLE
Price and location without information about size and "quality" is not helpful

### ISSUE THREAD

**Issue by memo42** [CodeforKarlsruhe/mietmap#7]
Price changes significantly with flat-size.
It also varies with "quality" of the flat meaning age of the house, features like a balcony and quality of the flooring, bathroom,...
Maybe look at the Mietspiegel issued by the city for more details.

A difference between the price calculated by the Mietspiegel and the real price might be a more helpful value to display. (Or a value based on the Mietspiegel that does not contain the district to make dispalying it on a map more useful)

In most cases immoscout does offer at least some of the information need.

**Comment by torfsen** [comment]
The price shown in the heat map is per m², so on a basic level the size is taken into account. Obviously the analysis could be more detailed, PRs in that direction are very welcome :wink:

AFAIK the Mietspiegel offered by the City of Karlsruhe is not available as open data, yet. It might be worth a try to issue a freedom of information request for the data via [FragDenStaat](https://www.fragdenstaat.de).

**Comment by memo42** [comment]
No, what I meant was that for a small (one room) flat the acceptable price per m² is significantly higher than for a larger flat.
